README.md

```diff
@@ -1 +1,3 @@
-# buildpack-rails-enable-node
+# buildpack-ruby-enable-node
+
+This is a buildpack that enables node at runtime for ruby apps.
```

Comment:
The idea was to use a BUILD env variable to either activate node at runtime or not. Could we add that here?

Reply:
added